src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialRoleMappingTest.java

```diff
@@ -87,11 +146,45 @@ void shouldReturnFeePaidRoles(String setOffice, String roleNameOutput) {
 
         //assertion
         assertFalse(roleAssignments.isEmpty());
-        assertEquals(1, roleAssignments.size());
-        assertEquals(roleNameOutput,roleAssignments.get(0).getRoleName());
         assertEquals(judicialOfficeHolders.stream().iterator().next().getUserId(),roleAssignments.get(0).getActorId());
-        assertEquals("Fee-Paid", roleAssignments.get(0).getAttributes().get("contractType").asText());
+        assertThat(roleAssignments.stream().map(RoleAssignment::getRoleName).collect(Collectors.toList()),
+                containsInAnyOrder(expectedRoles.split(",")));
+        assertEquals(expectedRoles.split(",").length, roleAssignments.size());
+        String regionId = allProfiles.iterator().next().getRegionId();
+        roleAssignments.forEach(r -> {
+            assertEquals("Fee-Paid", r.getAttributes().get("contractType").asText());
+            assertCommonRoleAssignmentAttributes(r, regionId, setOffice);
+        });
+    }
 
+    @ParameterizedTest
+    @CsvSource({
+            "SSCS Tribunal Judge-Fee Paid,'fee-paid-judge,judge,hmcts-judiciary'",
+    })
+    void shouldReturnFeePaidRolesWithBooking(String setOffice, String expectedRoles) throws IOException {
```

Comment:
Could this test be combined with the one above? a conditional to check the bits for with booking vs without? If we anticipate more scenarios being added to the shouldReturnFeePaidRolesWithBooking test in future then I am happy to leave it as is